### DISCUSSION TITLE
Error: EncryptedLocalStore internal error in macOs High Sierra/Catalina with captive runtime

### DISCUSSION THREAD

**Discussion by VaibhavPrajapati** [airsdk/Adobe-Runtime-Support#1225] [Q&A]
I have installed FLEX-SDK-4.6 from http://download.macromedia.com/pub/flex/sdk/flex_sdk_4.6.zip and files are overwritten by adobe air SDK for flex developer 33.1.1.575. 
The application is running properly under debug mode using ADL. But facing "Error: EncryptedLocalStore internal error" at runtime on screen.
I have tried with 33.1.1.575, 33.1.1.190, 33.1.1.127, 33.1.633 (latest)
(I'm not an expert in MacOS)

**Comment by ajwfrost** [comment]
Hi

Just to clarify: is the problem that you see this error when you're running it as a stand-alone application having packaged it as an app bundle? or is this happening when you run using ADL?

And are you running ADL from a terminal window, or is there an IDE involved?

If it's packaged up as an app bundle, it may be related to the security requirements or entitlements. Are you able to confirm what the call stack is for the error that's thrown?

thanks

**Comment by itlancer** [comment]
For me such issues fixed with latest AIR 51.1 versions.
https://github.com/airsdk/Adobe-Runtime-Support/issues/2253#issuecomment-2421677775
